README.md

```diff
@@ -17,6 +17,9 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pages: write 
```

Comment:
Thanks!

Is the `pages` permission necessary? It looks like that's only needed to create a site, request a pages build from the action, etc, but it seems to me that should happen automatically when the contents are written to.

Reply:
Hi. 

I have tested on my site and confirmed you are correct. It seems to be needed only when the user builds the site. And in this case, we don't need pages: permission. 

The following are just enough. 
```yaml
    permissions:
      contents: write
```

Reply:
I have re-submitted the pull request. Please kindly review.